make/TCP_Srv_Echo_silent.make

```diff
@@ -0,0 +1,135 @@
+# Alternative GNU Make project makefile autogenerated by Premake
```

Comment:
reduce makefiles to one

Reply:
Now we have 1 Makefile with several (3) configurations.

Works fine with gcc / make and VS.